clients/gtest/matmul_gtest.yaml

```diff
@@ -601,6 +601,24 @@ Tests:
   unit_check: 1
   gpu_arch: '94?'
 
+- name: matmul_f8_bf8_dst_fp16
+  category: pre_checkin
+  function:
+    matmul: *real_precisions_1b_dst_bf16
+  M: [128, 129]
+  N: [128, 129]
+  K: [128, 129]
+  transA: T
+  transB: N
+  alpha: 1
+  beta: [ 0.0, 2.0 ]
+  scaleA: [ 0, 1]
+  scaleB: [ 0, 1]
+  bias_vector: [0, 1]
+  bias_type: bf16_r
+  unit_check: 1
+  gpu_arch: '942'
+
```

Comment:
Call for suggestion: 942 only? or should I put yamls in 940 and 941 folders as well and make this '94?'

Reply:
By other backgroup reason, we need to make gfx942/gfx941 yamls identical. Therefore, you need to upload another yamls for gfx941.
I think it is fine to test it on 942 only. Just make sure 941 also have the solutions.

Reply:
I were wrong. We don't have gfx942 CI machine but only have 940/941. For CI testing requirement, you still need to add yamls for 940/941 and setthe gpu_arch as '94x' for ci test items.

Reply:
done